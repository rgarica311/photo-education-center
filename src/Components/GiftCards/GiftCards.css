.giftPhoto {
  background-image: url('http://creativephotoacademy.com/wp-content/uploads/2014/02/DSC_0402-edit-710x473.jpg');
  height: 25vh;
  position: relative;
  top: 80px;
  background-position: left;
  display: flex;
  align-items: center;
  justify-content: center;
  font-family: Raleway;
  font-weight: 900;
  font-size: 4rem;
  color: white;
}

.giftContainer {
  position: relative;
  top: 100px;
  padding: 20px;
}
