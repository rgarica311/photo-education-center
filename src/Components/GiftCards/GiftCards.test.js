import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import GiftCards from './GiftCards';

describe(`GiftCards Component`, () => {

  it('renders GiftCards by default', () => {
    const wrapper = shallow(<GiftCards />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
