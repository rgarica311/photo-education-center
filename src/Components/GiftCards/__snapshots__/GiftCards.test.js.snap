// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiftCards Component renders GiftCards by default 1`] = `
<Fragment>
  <div
    className="giftPhoto bannerPhoto"
  >
    GIFT CARDS
  </div>
  <div
    className="giftContainer"
  >
    <p>
      Great photographers make great images regardless of the tools they use. The latest and great camera is going to help you get better quality pictures but learning and discovering will help you make truly unique and compelling photographs.
    </p>
    <p>
      Gift cards are available in the following denominations online: $25, $49, $100, & $250  for any class.  If you would like to purchase a gift card in a different amount, they are also available in store or over the phone at 310-375-7014.
    </p>
    <p>
      When you purchase the gift card online, you will receive an email asking for a mailing address where we can ship the card, as well as the name of the recipient. The gift card package will include the card, a few class/workshop flyers, and a letter welcoming the recipient to Creative Photo Academy and explaining how to redeem the card for a class or workshop. Pricing below includes the cost of the card and $5.99 shipping and handling (includes insurance coverage on shipping up to $500).
    </p>
  </div>
</Fragment>
`;
